config/webpack.config.js

```diff
@@ -48,6 +48,13 @@ var config = {
       {
         test: /(\.png|\.gif)$/,
         loader: "url-loader?limit=32767"
+      },
+      {
+        test: /\.scss$/,
+        loader: ExtractTextPlugin.extract(
+          'style', // The backup style loader
```

Comment:
from webpack documentation:

> By convention, though not required, loaders are usually named as XXX-loader, where XXX is the context name. For example, json-loader.
> You may reference loaders by its full (actual) name (e.g. json-loader), or by its shorthand name (e.g. json).

personally i think the "-loader" appendix is redundent